<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManualWidget</class>
 <widget class="QMainWindow" name="UserManualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserManualWidget</string>
  </property>
  <property name="bks" stdset="0">
   <string>gray</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1312</width>
      <height>913</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>710</width>
      <height>680</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>913</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="BkLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>931</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="bks" stdset="0">
     <string>gray</string>
    </property>
   </widget>
   <zorder>BkLabel</zorder>
   <zorder>graphicsView</zorder>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>913</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="BtnWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>298</width>
         <height>88</height>
        </size>
       </property>
       <property name="bks" stdset="0">
        <string>white</string>
       </property>
       <widget class="QPushButton" name="chBtn">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>20</y>
          <width>120</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>生化</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="buttons" stdset="0">
         <string notr="true">left</string>
        </property>
       </widget>
       <widget class="QPushButton" name="imBtn">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>20</y>
          <width>120</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>免疫</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="buttons" stdset="0">
         <string notr="true">right</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="maximumSize">
    <size>
     <width>302</width>
     <height>913</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>302</width>
         <height>230</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QPushButton" name="ZoomInBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>83</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>放大</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ZoomOutBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>192</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>缩小</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QLabel" name="scaleLab">
        <property name="geometry">
         <rect>
          <x>93</x>
          <y>150</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>1%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="RecoverBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>262</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>恢复</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PrePageBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>459</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>上一页</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="NextPageBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>528</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>下一页</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="FirstPageBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>389</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>首页</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="LastPageBtn">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>598</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>末页</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="flat_return">
        <property name="geometry">
         <rect>
          <x>102</x>
          <y>770</y>
          <width>100</width>
          <height>44</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
       <widget class="QLabel" name="BkLabel2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>302</width>
          <height>931</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="bks" stdset="0">
         <string>white</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CurPageEdit">
        <property name="geometry">
         <rect>
          <x>102</x>
          <y>688</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="PageCountLab">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>678</y>
          <width>81</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>BkLabel2</zorder>
       <zorder>ZoomInBtn</zorder>
       <zorder>ZoomOutBtn</zorder>
       <zorder>scaleLab</zorder>
       <zorder>RecoverBtn</zorder>
       <zorder>PrePageBtn</zorder>
       <zorder>NextPageBtn</zorder>
       <zorder>FirstPageBtn</zorder>
       <zorder>LastPageBtn</zorder>
       <zorder>flat_return</zorder>
       <zorder>CurPageEdit</zorder>
       <zorder>PageCountLab</zorder>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
